accounts.go

```diff
@@ -40,9 +40,13 @@ type Account struct {
 
 type GetAccountInput struct{}
 
-func (client *AccountsClient) GetAccount(input *GetAccountInput) (*Account, error) {
+func (client *AccountsClient) GetAccount(ctx context.Context, input *GetAccountInput) (*Account, error) {
+	if ctx == nil {
+		panic(nilContext)
```

Comment:
Since I'm new to Go as well, how are other libraries migrating to this kind of API constraint with `context`? Are they providing defaults at points such as this (is that even good practice)?